app/profiles/queries/searchProfiles.ts

```diff
@@ -3,7 +3,7 @@ import db from "db"
 
 export default resolver.pipe(resolver.authorize(), async (search: String) => {
   const select = `
-    SELECT id as profileId, firstName || ' ' || lastName as name
+    SELECT id as profileId, firstName || ' ' || lastName || ' <' || email || '>' as name
```

Comment:
Show the email on the profile searches (on the selectbox) to make obvious that you can search by email too.